CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.15
+
+- Updated dependencies. ([#70](https://github.com/isoos/elastic_client/pull/70) by [SaadArdati](https://github.com/SaadArdati)).
```

Comment:
I think this should be about knn-search.

Reply:
Oh I'm on auto pilot, sorry haha